src/yt-music/YTMusicPlaybackControls.ts

```diff
@@ -11,7 +12,8 @@ export enum YTControl {
 }
 
 export class YTMusicPlaybackControls implements PlaybackControls {
-  constructor(private page: Page) {}
+
+  constructor(private page: Page, private SearchHandler: YTSearchHandler) {}
```

Comment:
Was this added by mistake?

Reply:
Was adding for earlier testing... I removed it? At least I though I did.